package.json

```diff
@@ -54,10 +54,12 @@
     "lint:js": "eslint --max-warnings=0 --ext=tsx --ext=ts --ext=js --ext=jsx --fix ./src",
     "start": "craco start",
     "build": "craco build",
+    "build:playwright-docker": "docker build --tag operations-gateway-playwright-test --file Dockerfile.playwright .",
     "serve:build": "yarn build && serve -l 5001 build",
     "test": "craco test --env=jsdom --coverage --watchAll=false",
     "test:watch": "craco test --env=jsdom --watch",
-    "test:playwright": "docker build --tag operations-gateway-playwright-test --file Dockerfile.playwright . && docker run --volume ./playwright-report:/app/playwright-report --publish 9323:9323 operations-gateway-playwright-test",
+    "playwright:test": "yarn build:playwright-docker && docker run --env LOCAL_DOCKER=true --volume ./playwright-report:/app/playwright-report --publish 9323:9323 operations-gateway-playwright-test",
+    "playwright:update-snapshot": "yarn build:playwright-docker && docker run --env LOCAL_DOCKER=true --volume .:/app --publish 9323:9323 operations-gateway-playwright-test",
```

Comment:
I tested that this does actually update snapshots - but how does it? I can't see any flags being passed to the playwright command...

Reply:
I passed in a different `--volume` flag for `playwright:test` and `playwright:update-snapshot`. Since updated snapshots are written inside the e2e folder, in order to write the changes to `e2e/` and `playwright-report/` folders to the host file system, I need to mount the entire project folder, hence `--volume .:/app`.

Reply:
Yeah, but doesn't playwright only update snapshots if you provide the `-u` flag?

Reply:
Good question, how did both of us manage to get the snapshots updating without the flag?? I will make another PR to add it back in.